### PR TITLE
Use ic-certified-assets crate from crates.io

### PR DESCRIPTION
**Pull request by jplevyak**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -4,24 +4,11 @@ version = "0.1.0"
...
-ic-cdk-macros = "0.2"
-ic-types = "0.1.1"
-ic-certified-map = { git = "https://github.com/dfinity/cdk-rs", rev = "7676e7f"}
-num-traits = "0.2.14"
-serde = "1"
-serde_bytes = "0.11"
-serde_cbor = "0.11"
-sha2 = "0.9.1"
+ic-certified-assets = "0.1.0"
+ic-cdk = "0.3.3"
```
Comment (lwshang) [inline Cargo.toml:13]: The latest `ic-cdk` and `ic-cdk-macros` are version 0.4.0.

Comment (jplevyak) [reply Cargo.toml]: Fixed.

**Review by lwshang — APPROVED** [review]

**Comment by 3cL1p5e7** [comment]
@jplevyak 
Excuse me, but where is the library source code?

**Comment by jplevyak** [comment]
@3cL1p5e7 dfinity/cdk-rs, but generally you can find the repo by searching crates.io https://crates.io/crates/ic-certified-assets

**Comment by 3cL1p5e7** [comment]
@jplevyak Thank you! <3

**Pull request merged**